filter_directories.py

```diff
@@ -13,7 +19,10 @@
     while directory:
         if os.path.exists( os.path.join(directory, '.shed.yml') ) or os.path.exists( os.path.join(directory, '.shed.yaml') ):
             if directory not in bl:
-                print(directory)
+                for filename in os.listdir(directory):
```

Comment:
fyi: @nsoranzo this will enhance the built-time for large PR like mothur. We will now split by xml file not per directory.
Need to get this into IUC as well.